README.md

```diff
@@ -3,36 +3,19 @@
 
 ## Prerequisites
 
-Verify that you have git installed on your system
-`git --version`
-
-Next, create a GitHub account if you do not have one:
-[GitHub Signup](https://github.com/signup)
-  
-Check to see if you have configured your git profile to match your GitHub user profile.
-
-After signing up, open a terminal and run the following git commands to set your name and email address (**Note** The email address must match the one you signed up with)
-
-`git config --global user.name "My Name"`
-
-`git config --global user.email "my@email.address"`
-
-## Listing of UFS and EPIC Repositories
-
-[UFS](https://github.com/ufs-community/ufs-weather-model)
-
-[Short Range Weather (SRW) App](https://github.com/ufs-community/ufs-srweather-app)
-
-[UFS Utilities](https://github.com/ufs-community/ufs_utils)
-
-[Land DA](https://github.com/ufs-community/land-DA_workflow)
-
-[JEDI](https://github.com/jcsda/fv3-jedi)
-
-[IODA](https://github.com/jcsda/ioda)
-
-[spack-stack](https://github.com/jcsda/spack-stack)
-
-[spack](https://github.com/jcsda/spack)
-
-[GSI](https://github.com/NOAA-EMC/GSI)
+1. Verify that you have git installed on your system: `git --version` (Note that version 2.30+ is recommended for working with UFS repositories; not required for today)
+2. Next, create a GitHub account if you do not have one: [GitHub Signup](https://github.com/signup)
```

Comment:
This could be rewritten to clarify by saying `... if you do not **already** have one:`.

Reply:
Great suggestion, I'll make that change.